frontend/src/constants.js

```diff
@@ -0,0 +1,2 @@
+export const ALPHA_API_KEY = "5DIHMIONQPZFEG6R";
+export const FINNHUB_API_KEY = "ciefl6pr01qmfas4d2hgciefl6pr01qmfas4d2i0";
```

Comment:
👍 
The next step is using an .env file like Diego mentioned
https://dev.to/felixdusengimana/environment-variables-in-reactjs-139l